src/Payments.sol

```diff
@@ -1664,6 +1664,70 @@ contract Payments is
 
         return result;
     }
+
+    /**
+     * @notice Gets information about an account - when it would go into debt, total balance, available balance, and lockup rate.
+     * @param token The token address to get account info for.
+     * @param owner The address of the account owner.
+     * @return fundedUntil The epoch at which the account would go into debt given current lockup rate and balance.
+     * @return totalBalance The total balance of the account.
+     * @return availableBalance The available balance (total balance minus current lockup).
+     * @return lockupRate The current lockup rate per epoch.
+     */
+    function getAccountInfo(
```

Comment:
this should be `simulateAccountSettlement` or `getAccountInfoIfSettled`

Reply:
Please also change all returned variable names accordingly. Especially `totalBalance ` and `availableBalance `.

Reply:
@Chaitu-Tatipamula Yeah makes sense.